pom.xml

```diff
@@ -200,225 +249,305 @@
     </build>
     <dependencies>
         <dependency>
+            <!-- Plexus Common Utilities -->
```

Comment:
Don't need / want this

Reply:
While I remember adding the comment for it, I do not remember adding the dependency to begin with, so I am fine with removing it.